lib/views/partials/loginform.html

```diff
@@ -4,28 +4,32 @@
       <h3>Brilliant Continuous Deployment.</h3>
       {% include "partials/_strider_copy.html" %}
 
-        <div class="row-fluid">
-            <div class="span4">
-                <img src="images/logo-250x250-transp.png" style="height: 192px" />
-            </div>
-            <div class = "span8">
-                <form class='login-form' action="/login" method="post">
-                    <div class="form-group">
-                        <label for="email"><strong>Email:</strong></label>
-                        <input type="email" class="form-control span12" id="email" name="email" placeholder="test@example.com"/>
-                    </div>
-                    <div class="form-group">
-                        <label for="password"><strong>Password:</strong></label>
-                        <input type="password" class="form-control span12" id="password" name="password"  placeholder="dontl00k"/>
-                    </div>
-                    <div>
-                        <input class="btn btn-success" type="submit" value="Log In"/>
-                    </div>
-                </form>
-                <p><strong><a href="/register">I have an invite code.</a></strong></p>
-                <p><a id="forgot-password-link" href="/forgot">Forgot your password?</a></p>
-            </div>
+      <div class="row-fluid">
+        <div class="span4">
+            <img src="images/logo-250x250-transp.png" style="height: 192px" />
+        </div>
+        <div class = "span8">
+          {% pluginblock LoginForm %}
+            <form class='login-form' action="/login" method="post">
+              <div class="form-group">
+                <label for="email"><strong>Email:</strong></label>
+                <input type="email" class="form-control span12" id="email" name="email" placeholder="test@example.com"/>
+              </div>
+              <div class="form-group">
+                <label for="password"><strong>Password:</strong></label>
+                <input type="password" class="form-control span12" id="password" name="password"  placeholder="dontl00k"/>
+              </div>
+              <div>
+                <input class="btn btn-success" type="submit" value="Log In"/>
+                {% pluginblock ExtraLoginButton %}
+                {% endpluginblock %}
+              </div>
+            </form>
+            <p><strong><a href="/register">I have an invite code.</a></strong></p>
+            <p><a id="forgot-password-link" href="/forgot">Forgot your password?</a></p>
+          {% endpluginblock %}
         </div>
       </div>
-   </div>
+    </div>
+  </div>
```

Comment:
Apologies for the whitespace diffs here - I tidied it up a bit to use 2-space indents; it had been mixed 2-space and 4-space.